project_national/national.yml

```diff
@@ -1,11 +1,13 @@
 stock_type: residential
 buildstock_directory: ../../OpenStudio-BuildStock # Relative to this file or absolute
 project_directory: project_national # Relative to buildstock_directory
-output_directory: /scratch/<username>/outputs/project_national
-weather_files_path: /shared-projects/buildstock/weather/project_resstock_national_weather.zip  # Relative to this file or absolute path to zipped weather files
+output_directory: /scratch/<username>/outputs/project_national # Or specify local path on computer
+weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip # TMY3 files can be found at the NREL Data Catalog page using this path
```

Comment:
This is publicly accessible? And you've confirmed that it works?

Reply:
It is publicly accessible but I haven't tested it on eagle yet. I just tried to but keep getting an 'access denied' error which seems to be a problem with the data.nrel.gov website (@nmerket confirmed this). I will post an update as soon as they resolve the problems.

Reply:
@joseph-robertson the files are publicly accessible and have been tested and works well.